### PR TITLE
rename: to kalix

### PR DESCRIPTION
**Pull request by octonato**
Depends on 

* kalix-action - https://github.com/lightbend/kalix-action/pull/1
* kalix-framework release - for the annotations and proxy container
* kalix-javascript-sdk release

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 version: "3"
 services:
   akka-serverless-proxy:
-    image: gcr.io/akkaserverless-public/akkaserverless-proxy:0.7.0-beta.10
-    command: -Dconfig.resource=dev-mode.conf -Dakkaserverless.proxy.eventing.support=google-pubsub-emulator
+    image: gcr.io/kalix-public/kalix-proxy:0.7.0-beta.10
```
Comment (octonato) [inline docker-compose.yml]: also time to update the version here, but we can wait until we have 0.9.0

Comment (ennru) [reply docker-compose.yml]: Add a note to the proxy release instructions about this version.

Comment (octonato) [reply docker-compose.yml]: What do you mean? You mean so we don't forget to update it here? 

This is more an example app. Keeping things in sync on each release adds lots of overhead. We may prefer to add it as a sample in the javascript sdk instead and ensure that it in sync with all other samples. 

(btw, version 0.9.0 was already released before the renaming)

Comment (ennru) [reply docker-compose.yml]: Right, maybe not worth caring about upgrading this.

**Comment by octonato** [comment]
Putting it back to draft for now. It seems that in the renaming document we were planning to create a new repo for this instead of renaming here. I will check tomorrow if that's still the case.

**Comment by ennru** [comment]
I agree, leave this pending until we have the bandwidth to take responsibility for it.

**Comment by octonato** [comment]
There is a page in the docs where we point to this example. We will need to remove it and have something similar to what we have for CircleCI. 

See https://docs.kalix.dev/projects/integrate-cicd-github-actions.html#_example_workflow

**Comment by octonato** [comment]
I created an issue in the docs repo about it: 
https://github.com/lightbend/akkaserverless-docs/issues/1208

**Comment by ennru** [comment]
We decided to keep it. Let's create the Kalix taste of this in a new repository.

**Comment by octonato** [comment]
closing this since obsolete

**Pull request closed without merging**